sycl/plugins/opencl/pi_opencl.cpp

```diff
@@ -1993,6 +1993,7 @@ pi_result piPluginInit(pi_plugin *PluginInit) {
   // Queue
   _PI_CL(piQueueCreate, piQueueCreate)
   _PI_CL(piextQueueCreate, piextQueueCreate)
+  _PI_CL(piextQueueCreate2, piextQueueCreate)
```

Comment:
why don't you need to "die" for other "2" APIs?

Reply:
piQueueCreate/piQueueCreate2 are supposed to be used for SYCL queues, and all plugins will need to support them.
The make_queue2 and get_native2 are only needed in the L0 plugin, so they are left out of this plugin.

Reply:
I think you should still "die" in those other plugins as unsupported

Reply:
Ok, done.